packages/eos-components/src/components/CarouselCard.vue

```diff
@@ -138,9 +138,13 @@ export default {
 
 .card-content .head {
   min-height: 3 * ($h2-font-size * $headings-line-height);
+  max-width: calc(100% - 100px);
```

Comment:
I guess we can use a flexbox to do this properly and avoid hardcoding the button size, which will change depending on the text label. Think if in the future we support translations.